components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -588,9 +589,13 @@ bool BravePrefProvider::SetWebsiteSettingInternal(
       // brave::SetDefaultThirdPartyCookieBlockValue).
       return false;
     }
+    if (content_type == ContentSettingsType::BRAVE_HTTP_UPGRADABLE_RESOURCES) {
```

Comment:
it seems like we're missing a content settings migration here for the default setting

Reply:
Added migration, thanks!